packages/starlight/components/SidebarSublist.astro

```diff
@@ -15,10 +16,13 @@ interface Props {
             {entry.label}
           </a>
         ) : (
-          <>
-            <h2>{entry.label}</h2>
+          <details open>
+            <summary class="sidebar-group-header">
+              <h2>{entry.label}</h2>
+              <Icon name="down-caret" class="icon caret" />
```

Comment:
The icons are also 20px in the designs, so we can make these a bit larger and I’m going to suggest using `right-caret` here so we can simplify the CSS selector:

```suggestion
              <Icon name="right-caret" class="caret" size="1.25rem" />
```

(Also removed the `icon` class as it’s not used here.)